jest.config.ts

```diff
@@ -124,11 +124,15 @@ export default {
     },
     {
       displayName: 'client-e2e',
-      testEnvironment: 'jsdom',
+      testEnvironment: 'node',
```

Comment:
A bit annoying/surprising, but these tests actually work in node with the latest changes, and DON'T work in jsdom. This is because the `jsdom` test platform is actually missing some of the newest web-standards which recent node versions actually have (`fetch`, `TextEncoder`, etc).